### PR TITLE
Clarified the self-bot warning

### PR DESCRIPTION
**Pull request by erkinalp**
Self-bots can also be detected when they use bot token, just harder

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,8 +85,6 @@ your.token-here`.
 
 ### Linking your Discord account
 
-**Warning**: Linking your user account's token is against Discord's Terms of Service.
```
Comment (Sorunome) [inline README.md:84]: It feels better to still leave this warning here explicitly. Or stating that linking a discord account is a self-bot which was mentioned above to be against ToS

File: README.md [hunk 2]
```diff
@@ -44,6 +44,10 @@ npm run start
 Start a chat with `@_discordpuppet_bot:yourserver.com`. When it joins, type
 `help` in the chat to see instructions.
 
+**Warning**: Self-bots are against Discord's Terms of Service.
+* When used with user tokens, you might have your account disabled or terminated.
+* When used with bot tokens, you might lose ability to add new bots or have your account terminated.
```
Comment (Sorunome) [inline README.md:49]: Where do you get this from? this is the first time soru is hearing issues with bot tokens.

Comment (erkinalp) [reply README.md:49]: I asked at DTesters and they said using a bot in a human-like manner is also against ToS. Discord Bot Client is a famous example of what happens when you use bot using a human-targeted client in a human-like manner.

Comment (erkinalp) [reply README.md:49]: And they also said they would do the same even if you did the same thing using a bot developed from scratch.

**Comment by dsonck92** [comment]
Tracked by: https://gitlab.com/mx-puppet/discord/mx-puppet-discord/-/merge_requests/110

**Pull request closed without merging**